app/Http/Controllers/BeatmapPacksController.php

```diff
@@ -43,6 +97,14 @@ public function show($idOrTag)
         $sets = $pack->beatmapsets;
         $userCompletionData = $pack->userCompletionData(Auth::user());
 
+        if (is_api_request()) {
+            return json_item(
+                $pack,
+                new BeatmapPackTransformer($userCompletionData),
+                ['beatmapsets', 'user_completion_data']
+            );
+        }
```

Comment:
api should use `cursor_string` instead of page for pagination, see `EventsController` for an example